content/tutorials/reactjs-realtime-commenting.textile

```diff
@@ -227,44 +231,66 @@ addComment(e) {
 
     // Clear input fields
     e.target.elements.comment.value = '';
-    e.target.elements.name.value = '';
   }
 }
-```
+div.
 
-We first prevent the default form submission behaviour (that is, reloading the page). We then get the form inputs entered (name and comment) and make sure they are not just some empty strings. If the inputs are properly filled, we add the comment just made to the array of comments. Then we publish the comment just made to a @comments@ channel with an @add_comment@ event. Adding @/*global Ably*/@ tells ESlint that we are using the global variable (@Ably@) we defined earlier. With this, we won't get the Eslint error: @'Ably' is not defined  no-undef@. Lastly, we clear the form fields.
+We first prevent the default form submission behaviour (that is, reloading the page). We then get the form inputs entered (name and comment) and make sure they are not just some empty strings. If the inputs are properly filled, we add the comment just made to the array of comments. Then we publish the comment just made to a @comments@ channel with an @add_comment@ event. Lastly, we clear the form fields.
 
 Next, let's bind @addComment()@ to the @this@ keyword by adding this line to the @constructor()@ just after @super()@:
 
-```[javascript]
+bc..
 this.addComment = this.addComment.bind(this);
-```
+div.
 
-"See this step in Github":https://github.com/ably/tutorials/tree/reactjs-commenting-step9
+Also we need to import @Ably@, to do so, add following line to the top of the file below other import commands:
+
+bc..
+import Ably from './Ably';
+div.
+
+"See this step in Github":https://github.com/marsielko/tutorials/tree/4689eef90b22bdfafedc1b60d18dbdeae128e7d7
 
 h2. Step 10 - Create Comment Component
 
-Within the @components@ folder, create a new file named @Comment.js@ and add the code below into it:
+Within the @src/components@ folder, create a new file named @Comment.js@ and add the code below into it:
 
-```[javascript]
+bc..
 import React, { Component } from 'react';
 
 class Comment extends Component {
+  constructor(params) {
+    super(params);
+    this.messageDate = this.messageDateGet();
+  }
+
+  messageDateGet() {
+    const date = new Date(this.props.comment.timestamp);
+    const dateTimeFormatOptions = {
+      "year": '2-digit',
+      "month": '2-digit',
+      "day": '2-digit',
+      "hour": '2-digit',
+      "minute": '2-digit'
+    };
+    const localeString = date.toLocaleString(undefined, dateTimeFormatOptions);
+
+    return localeString;
+  }
+
   render() {
     return (
       <article className="media">
         <figure className="media-left">
           <p className="image is-64x64">
-           <img src="https://bulma.io/images/placeholders/128x128.png" alt="Avatar" />
+            <img src="https://bulma.io/images/placeholders/128x128.png" alt="Avatar" />
```

Comment:
maybe we should create a local asset. something we can chat about on Thursday.
![](https://bulma.io/images/placeholders/128x128.png) I think we can make something nicer? wdyt?

Reply:
Definitely, this one looks terrible. :)

Reply:
@marsielko - are you able to address this issue?